src/CMakeLists.txt

```diff
@@ -192,7 +202,13 @@ else(INTEL_SSE4_1)
 endif(INTEL_SSE4_1)
 
 
-set(EXTRALIBS uuid rt dl ${ATOMIC_OPS_LIBRARIES})
+set(EXTRALIBS rt ${ATOMIC_OPS_LIBRARIES})
+if(${LINUX})
+  set(LIB_DL dl)
```

Comment:
we should switch to `CMAKE_DL_LIBS`. see https://cmake.org/cmake/help/v2.8.11/cmake.html#variable:CMAKE_DL_LIBS